Description: add the actual binary names from the qemu packages so default
 hypervisor detection works properly on first run.
Author: Marc Deslauriers <marc.deslauriers@canonical.com>

--- a/virtManager/connect.py
+++ b/virtManager/connect.py
@@ -115,6 +115,8 @@
         if (os.path.exists("/usr/bin/qemu") or
             os.path.exists("/usr/bin/qemu-kvm") or
             os.path.exists("/usr/bin/kvm") or
+            os.path.exists("/usr/bin/qemu-system-x86_64") or
+            os.path.exists("/usr/bin/qemu-system-i386") or
             os.path.exists("/usr/libexec/qemu-kvm") or
             glob.glob("/usr/bin/qemu-system-*")):
             return "qemu:///system"
